prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -78,6 +78,14 @@ class FhirToHl7Converter(
         // Sanity check, but at this point we know we have a good schema
         check(message != null)
         terser = Terser(message)
+        val dupes = (
+            schemaRef.elements.filter { it.name != null } +
+                schemaRef.elements.flatMap { it.schemaRef?.elements ?: emptyList() }
+            ).groupingBy { it.name }.eachCount().filter { it.value > 1 }
+
+        if (dupes.isNotEmpty()) { // value is the number of matches
+            throw SchemaException("Schema ${schemaRef.name} had multiple elements with the same name: $dupes")
```

Comment:
Change to: Schema x HAS ...

Reply:
The $dupes portion shows this: Schema ORU_R01-base had multiple elements with the same name: {set-id=2}
What is set-id=2?  Can we cleanup that last portion to make it more readable?  Not sure what the 2 is.

Reply:
Updated it to be clearer